src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -3,37 +3,67 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
+import edu.kis.powp.command.ComplexCommandGenerator;
+import edu.kis.powp.command.ShapeFactory;
 import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.drivers.JaneFiguresDriver;
 import edu.kis.powp.jobs2d.drivers.adapter.AbstractAdapter;
 import edu.kis.powp.jobs2d.magicpresets.FiguresJane;
 import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
 
 public class SelectTestFigureOptionListener implements ActionListener {
 
-	private DriverManager driverManager;
-
-	public SelectTestFigureOptionListener(DriverManager driverManager) {
-		this.driverManager = driverManager;
-	}
-
-	@Override
-	public void actionPerformed(ActionEvent e) {
-		switch (e.getActionCommand()) {
-			case "Figure Joe 1": {
-				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-				break;
-			}
-			case "Figure Joe 2": {
-				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
-				break;
-			}
-			case "Figure Jane": {
-				FiguresJane.figureScript(new AbstractAdapter(driverManager.getCurrentDriver()));
-				break;
-			}
-			default: {
-				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-			}
-		}
-	}
+    private DriverManager driverManager;
```

Comment:
The original formatting convention used tabs (not spaces). Please be more careful.